devicemodel/include/vga.h

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright (C) 2022 Intel Corporation. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause-FreeBSD
```

Comment:
Same comment about the license tag, this one is deprecated.

Reply:
I feel we don't have to update a deprecated license to a newer one, especially when it was from 3rd party we leveraged. New projects should not use deprecated license tag, but the license of the older projects should be still effective.

Reply:
Thanks for your input @yaomichael , should we have kept the original copyright notice in there too then (alongside the Intel copyright). Where was this leveraged from?

Reply:
good point... just checked and vga.h was from FreeBSD/bhyve. We need to add original copyright notice back

Reply:
> I feel we don't have to update a deprecated license to a newer one, especially when it was from 3rd party we leveraged. New projects should not use deprecated license tag, but the license of the older projects should be still effective.

Hi @yaomichael , just as FYI, this is indeed in line with the legal team's guidance. Thanks for your input on this!